tests/url.rs

```diff
@@ -200,10 +208,7 @@ fn multiple() {
 
 #[test]
 fn international() {
-    assert_linked(
-        "http://üñîçøðé.com/ä",
-        "|http://üñîçøðé.com/ä|",
-    );
+    assert_linked("http://üñîçøðé.com/ä", "|http://üñîçøðé.com/ä|");
```

Comment:
This comes from `cargo fmt`. I kept it because it arguably looks more similar to the other checks.